ERCS/erc-7888.md

```diff
@@ -0,0 +1,454 @@
+---
+eip: 7888
```

Comment:
```suggestion
eip: 7838
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.